sources/packages/backend/libs/utilities/src/config/config.models.ts

```diff
@@ -1,4 +1,9 @@
-export type ClientType = "student" | "institution" | "aest" | "supportingUsers";
+export type ClientType =
+  | "student"
+  | "institution"
+  | "aest"
+  | "supportingUsers"
+  | "external";
```

Comment:
I am not sure this one should be exposed in the API configs.
Was there a need for it or was it added for consistency?

Reply:
Yes, we don't need that. Thanks!